gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -456,6 +461,29 @@ gst_tensor_filter_transform (GstBaseTransform * trans,
       GST_STR_NULL (prop->model_files[0]));
   allocate_in_invoke = gst_tensor_filter_allocate_in_invoke (priv);
 
+  /* skip input data when throttling delay is set */
+  if (priv->throttling_delay != 0) {
```

Comment:
As per our offline discussion, this would be best applied to tensor_converter and tensor_src*

Reply:
Yes, sure. I will make next PRs for that.

Reply:
What if...
```
t.c (doing 100FPS) --> T -+-> T-F model A (may do 10 FPS)
                          +-> T-F model B (may do 200 FPS)
```
Do you still want to handle overflow QoS events at t.c?

Reply:
Yes, I have to. But, this PR is targeting to compare its latency and throttling delay from `tensor_rate`, as a prototype.
I will make another PR to consider multiple QoS events from downstream elements.

Reply:
@dongju-chae I wonder if the images in the PR description have been updated or not? Because the fps of tensor_converter is 36FPS. However, with this change, we expect it to also be 5fps.
Please correct me if I understood wrongly.

Reply:
Yep, it seems like a glitch. It keeps slightly changing over time.